andela/titan-dev-ah-frontend
# #163518655 Share the article via email, facebook, and twitter

#### What does this PR do?

- It adds the ability to share an article via facebook, twitter, and email
#### Description of Task to be completed?

- There is a button for each facebook, twitter, and email on `single article page` that helps the user to share that specific article. 
- When the user clicks on one of these buttons described above, they get redirected to a specific page where they compose their post to share including the link to that specific article being shared.

#### How should this be manually tested?
- After cloning the repository, check out `t-share-article-via-social-media-163518654` branch
- Run `yarn` to install the project dependencies
- Run `yarn start` to start the application then, you navigate to any of the articles
- Try navigating to a single article page `/articles/<article slug>`
- if you click on any of the left buttons for sharing, you should be directed to the relevant page to customize your post depending on which platform you want to share to:(email, facebook, and/or twitter)

#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- #163518654
